src/reducers/appLifecycle.js

```diff
@@ -1,10 +1,10 @@
-import {ACTIONS} from '../actions/constants';
-import {APP_STATES} from './constants';
+import {getAction} from '../actions/constants';
+import {getAppState} from './constants';
 
-function appLifecycle(state=APP_STATES('STARTED'), action) {
+function appLifecycle(state = getAppState('STARTED'), action) {
```

Comment:
Why was this changed ? Is it part of the prettier update ? 
I am not sure they should be renamed as they are meant to be constants and in regular redux pattern the `ACTIONS` are always capitalized.

As a side note, I don't think the ACTIONS/APP_STATES should be functions, it only send an error if there is no action defined, that is only useful when developing since dispatching invalid actions is not something that should happen in a production app. You also don't get autocompletion in IDE with the function approach, meaning it is easier to make typos and get that error.

Not related, but I find the three different constants files a bit confusing.

Reply:
This was changed because it threw a linting error, stating that functions shouldn't be capitalized unless it's a constructor. I agree with you that they shouldn't be functions to begin with, though! I was planning on doing a bigger refactor once this is merged, as part of the loading states PR I'm working on. For now, I would like to merge this (if it won't cause issues) so that I can move on with bigger refactor stuff - this at least stops the linting errors.